pkgs/applications/networking/p2p/stig/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, fetchFromGitHub
+, python
+}:
+
+with python.pkgs; buildPythonApplication rec {
```

Comment:
I've found that example from somewhere in Nixpkgs but your argument makes sense.

Reply:
Nixpkgs is old; Language and preferences have shifted over time. Also unfortunately different reviewers sometimes have different opinions.

So what I'm saying is you can probably find at least one example of just about anything in nixpkgs ;) If there is a convincing argument for or against something, you should do that. Otherwise, go with the majority of nixpkgs.